debezium-core/src/main/java/io/debezium/config/CommonConnectorConfig.java

```diff
@@ -637,6 +640,20 @@ public static FieldNameAdjustmentMode parse(String value) {
                     + " If set a stacktrace of non-retriable exception is traversed and messages are"
                     + " matched against this regular expression. If matched the error is changed to retriable.")
             .withDefault(false);
+    public static final Field NOTIFICATION_TOPIC = Field.create(NOTIFICATION_CONFIGURATION_FIELD_PREFIX_STRING + "topic.name")
```

Comment:
Please change the prefix to `notification.sink` for this option.

As the follow-up - the notification channel specific field definiitons should be moved to the channel if possible.